content/en/contribute/medic/product-development-process/code-health.md

```diff
@@ -6,15 +6,42 @@ description: >
   How to keep the code and systems up-to-date
 ---
 
-The Code Health process is intended to ensure we are being responsible about keeping our code and system up-to-date and “healthy” while we focus on work via OKRs.
+Healthy code is code that is easy to understand and update. Over time all code becomes less healthy as dependencies become out of date, new language features or code style rules are introduced, and changes are made that add complexity. This is sometimes referred to as technical debt as it makes it progressively harder to make new code changes. No project is ever completely healthy, so we need an ongoing process to improve the least healthy parts of the code so the CHT can be sustainable long term.
 
-Work on code health is done via tickets in GitHub. There are plenty of times when engineers can fix or improve code before getting back to other tasks. That works great for minor fixes but may lead to postponing complex tasks. To ensure we don't neglect important, but more complex, code health tasks these will be scheduled by each PM/PO on each Focused Working Group. This allows the Focused Working Group to stay on track with their regular work, but then take on code health tasks when the time is right. It is important for each PM/PO to work with leadership to make time for this to happen.
+A good way to differentiate code health improvements from other changes is if it's something that a user won't notice, then it's code health. Because no user will ever ask for the change to be made they need to be managed outside of the usual prioritisation process.
 
+A good guideline is to spend 25% of your time improving code health, which can be split up into three distinct categories: Tidying, Cards, and Projects.
 
-### An example of how this works in practice
+### Tidying
+
+These are the smallest improvements and should be part of your work every week. Engineers can decide what tidying they want to prioritize and don't need any approval to do the work, but it will need to go through [Code Review]({{% ref "/contribute/code/workflow#code-reviews" %}}) to ensure a teammate agrees it's an improvement. Tidying can be done as part of an issue you're working on, or you can create a new issue and make the change as a separate PR, depending on which you prefer.
```

Comment:
> Tidying can be done as part of an issue you're working on, or you can create a new issue and make the change as a separate PR, depending on which you prefer.

My only concern here is when the tidying work is outside of the scope of the current branch/issue, it might make the code review more difficult to review, as the reviewer will have to review each commit separately to understand which change is tidying and which change is the actual feature. 

Long story short, I recommend creating a new issue for tidying changes to reduce PR complexity and the risk of breaking things (e.g. via a dependency upgrade).

Reply:
I've updated the text to cover this in more depth, thanks!